scripts/Hirose/df13_angled_pth.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+import sys
+sys.path.append(r'''C:\kicad\fp-gen\kicad_mod''')
```

Comment:
this is an absolute path, which causes problems on every computer with another file hierarchy or non Windows Operating System.

Does my default code for loading the correct directory via relative path not working on Windows?

```python
sys.path.append("../../kicad_mod") # load kicad_mod path
```